src/participant/participant.ts

```diff
@@ -188,10 +191,28 @@ export default class ParticipantController {
         'workbench.action.chat.clearHistory'
       );
     }
+    const commandPrefix = command ? `/${command} ` : '';
+    const query = `@MongoDB ${commandPrefix}${message}`;
+
+    if (telemetry) {
```

Comment:
This is an example of it might be nice to structure tracking in other functions as some hot-pluggable thing.